src/cmd/create.rs

```diff
@@ -31,15 +40,16 @@ pub struct Basic {
     force: bool,
 
     #[structopt(long)]
-    /// Use space separated seed words to create the wallet
-    seed: bool,
+    /// Use a BIP39 or mobile app seed phrase to generate the wallet keys
+    /// (bip39, mobile)
+    seed: Option<SeedType>,
 
     #[structopt(long, default_value = NETTYPE_MAIN_STR)]
     /// The network to generate the wallet (testnet/mainnet)
     network: Network,
 
     #[structopt(long, default_value = KEYTYPE_ED25519_STR)]
-    /// The type of key to generate (ecc_compact/ed25519(.
+    /// The type of key to generate (ecc_compact/ed25519)
```

Comment:
Not related to my change, but noticed it (and the similar one below) while making changes here.